exampleCourse/questions/templateRandomMultipleChoiceWithFigure/server.py

```diff
@@ -0,0 +1,22 @@
+import random
+
+def generate(data):
+
+    #Create a list with the file name of all UIUC figures
+    allUIUCfigures = ["fig"+str(i)+".jpeg" for i in range(1,8)]
+
+    #Make a random selection of 3 figures from UIUC
+    trueFig = random.sample(allUIUCfigures,3)
+
+    #Store the true figures in data['params']
+    for i,fig in enumerate(trueFig):
+        data['params']['filename'+str(i+1)] = fig
+
+    #Create a list with the file name of all non-UIUC figures
+    allnonUIUCfigures = ["b"+str(i)+".jpeg" for i in range(1,5)]
+
+    #Make a random selection for the non-UIUC figure
+    falseFig = random.choice(allnonUIUCfigures)
+
+    #Store the false figure in data['params']
+    data['params']['filename'] = falseFig
```

Comment:
I found the naming confusing here, because the `falseFig` is the one that's the _correct_ answer. Could we name the figures something like `data['params']['correct_filename']` and `data['params']['wrong_filename1']`?

Where `correct_filename` is the non-UIUC fig.